HotTips/Engine/VSTipHistoryManager.cs

```diff
@@ -60,27 +72,41 @@ private string GetTipHistoryFromSettingsStore()
             return tipHistoryRaw;
         }
 
-        public void MarkTipAsSeen(string globalTipId)
+        public void MarkTipAsSeen(TipHistoryInfo globalTipId)
```

Comment:
Variable name no longer matches the type.

Reply:
Also, it looks like we only need the globalTipId in this method. Was there a reason to change the input parameter?